E/EllipsisNotation/Compat.toml

```diff
@@ -11,4 +11,4 @@ julia = "1.5.0-1"
 ArrayInterface = "3"
```

Comment:
then this would need to get changed to 1.1.1

Reply:
Shouldn't we change the bounds of the last entry to "1.1.3-1" instead, because 1.1.2 was the last version where we needed to be below ArrayInterface=3.2

Reply:
No, the issue is that it requires being above 3.2. That's why 1.1.3 was made.